about.html

```diff
@@ -0,0 +1,192 @@
+        </li>
+        <li>
+          <a href="#" class="mobile__nav__links">Program</a>
+        </li>
+        <li>
+          <a href="#" class="mobile__nav__links">Join</a>
+        </li>
+        <li>
+          <a href="#" class="mobile__nav__links">Sponsor</a>
+        </li>
+        <li>
+          <a href="#" class="mobile__nav__links">News</a>
+        </li>
+      </ul>
+    </nav>
+    <!---------------------------- End Here -------------------->
+    <section class="about">
+      <h3 class="about__title">
+        Hello! United Nation Climate Change Conference
+      </h3>
+      <h1 class="about__subtitle">
+        Sharm el-Sheikh Climate Change Conference - November 2022
+      </h1>
+      <hr class="short__hr">
+      <p class="about__description">
+        At the Sharm el-Sheikh Climate Change Conference (COP 27), countries
+        came together to take action towards achieving the world's collective
+        climate goals as agreed under the Paris Agreement and the Convention.
+        The conference took place from 6-20 November 2022 in Sharm el-Sheikh,
+        Egypt. Heads of State and Government attended the Sharm el-Sheikh
+        Climate Implementation Summit on 7 and 8 November. A high-level segment
+        primarily attended by Ministers took place from 15-18 November.
+      </p>
+      <h2 class="contact">
+        Please contact us per Email for any further questions about Sharm
+        el-Sheikh Climate Change 2022!
+      </h2>
+      <a href="#" class="about__email">ungc@unisef.org</a>
+
+    </section>
+
+    <section class="about__logo">
+      <h3 class="about__logo__title">Creating UN Climate Change 2022 Logo</h3>
+      <hr class="short__hr" />
+      <p class="about__logo__description">
+        The logo of UN CC conference 2022 was decided through the logo
+        competition from 8. June to 7. July
+      </p>
+      <img class="about__img" src="./images/logo.png" alt="United Nation Logo" />
+    </section>
+
+    <section class="past__conference">
+      <h3 class="past__title">See the past UN Climate Change Conference</h3>
+      <hr class="short__hr" />
+      <p class="past__description">
+        Take a look at the last two UN Climate Change Conferences which place in
+        Buenos Aires and in Warsaw
+      </p>
+      <div class="past__cards">
+        <div class="past1">
+            <div class="overlay">
+                <div class="past__card__content">
+                    <h3 class="past__year">2013</h3>
+                    <p class="past__description__img">UN Conference 2013</p>
+                </div>
+            </div>
+        </div>
+        <div class="past2">
+            <div class="overlay">
+                <div class="past__card__content">
+                    <h3 class="past__year">2022</h3>
+                    <p class="past__description__img">UN Conference 2022</p>
+                </div>
+            </div>
+        </div>
+      </div>
+    </section>
+
```

Comment:
- [ ] I think your past events cards are occupying too much width, kindly reduce to stick to Cindy's design :recycle: 

![Screenshot from 2023-03-16 08-56-10](https://user-images.githubusercontent.com/97834160/225539406-ef43591d-012c-42b4-9e5f-ec12f2941ef3.png)

Reply:
Done